.github/workflows/aws-test-frontend-role.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  release:
+    types: [created]
+
+name: Test Role
+
+jobs:
+  deploy:
+    name: Test Frontend Role Configuration
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      
+    - name: Get bot token from Parameter Store 
+      uses: marvinpinto/action-inject-ssm-secrets@latest 
+      with: 
+        ssm_parameter: /path/to/parameter/AWS_FRONTEND_NONPROD_ROLE 
```

Comment:
This wouldnt be the right parameter path, would it? Did you put the parameter in parameter store?